env-node.js

```diff
@@ -13,9 +13,8 @@ module.exports = {
 	'rules': {
 		'node/no-deprecated-api': 'warn',
 		'node/no-unpublished-bin': 'warn',
-		'node/no-unsupported-features/es-builtins': 'warn', // TODO: Error in 5.0.0
```

Comment:
Are these "disabled" or "error" by default? if this is changing to error it should be in a major bump instead, otherwise ignore.

Reply:
All I did was remove the TODO

Reply:
Ah ok i see these have been deprecated. I was worried if they were set to "error" by the `node` plugin by default and this config was overriding it by setting it to warn. By removing this line we'd be breaking clients by enabling the rule inadvertently.